conduct/FOUNDATION_CODE_OF_CONDUCT_REQUIREMENTS.md

```diff
@@ -44,28 +44,32 @@ in place by a project (for reports made to a project) or the CPC (in the
 case of reports relating to spaces within the scope of the foundation versus a project).
 In exceptional cases where a reporter wishes to challenge the response from the CPC or
 project, or does not feel comfortable reporting through the standard processes, an escalation
-report can be made to coc-escalation@openjsf.org.
+report can be made to report@openjsf.org.
 
 All escalations will be reviewed and investigated by the Code of Conduct Panel (CoCP)
 and will result in a response that is deemed necessary and appropriate to the circumstances.
 The Code of Conduct panel is obligated to maintain confidentiality with regard
 to the reporter of an escalation.
 
-### Code of Conduct panel
+### Code of Conduct Panel
 
-The Code of Conduct panel is a cross-foundation team who have been trained in moderation
+The Code of Conduct Panel is a cross-foundation team who have been trained in moderation
 and code of conduct issues. The makeup of this team is as follows:
 
-* The Foundation Executive director
-* 1 member from the Board
-* 1 member from the CPC
-* 1 member of the OpenJS marketing committee
-* each top-level project may optionally provide 1 member
-* the non top-level projects may optionally provide 1 member
-* optionally 1 invited outside expert as agreed by the other CoCP members.
+* OpenJS Foundation Executive Director - Robin Ginn (@rginn)
+* OpenJS Foundation Marketing Lead - Kylie Wagar-Dirks (@kyliewd)
+* 3 members selected by the Cross Project Council
+  - TBD
+  - TBD
+  - TBD
 
-Members of the CoCP team will be added for a 1-year term and will be re-confirmed on
-a yearly basis.
+#### Nomination
+
+Code of Conduct Panel team members are CPC Members who self-nominate or are community
+members nominated by the CPC. CoCP team members must be approved by the CPC and re-confirmed
+every year. If there are no objections after seven days, the nomination is automatically
+accepted. If there are objections to a specific nomination, then a simple majority vote of
+the CPC in favor of the nomination is required.
```

Comment:
I'm uneasy with this. If the objection is substantiated (e.g. the person has seriously violated the CoC in the past, or is known to hold racist or sexist positions publicly, for example), they can't sit in the CoC team regardless of whether or not they'd be voted in.

Not sure what the best solution is, here. Maybe the existing CoC team could decide whether to confirm or overrule the objection?

Reply:
I think if no consensus is reached, the member doesn't get approved? I haven't re-read the text (on a phone rn) but if it's not explicit enough, we should make it.

Reply:
> I think if no consensus is reached, the member doesn't get approved

So that's precisely what I'm asking for, but it's neither what simple majority voting (what is currently proposed in the PR) nor what STV (what is currently proposed in an edit above) does.

Reply:
In the Node.js project, where there is no voting, there bar is that there are no objections. Perhaps we should merge the two approaches as such: a vote happens, but if there is an objection to any nomination, that person will not be eligible.

Reply:
I still don't understand why we should be voting in the first place.

Reply:
I like the "moderators self nominate and there must be no objections from the leadership body or bodies that get moderated, recertification once a year" process since:

 - It holds moderators to high standards of being relatively objective when elected.
 - The leadership body needs to vote to remove moderators during their term but no single member can remove a moderator after elected not during re-certification which helps in cases a leadership body member is themselves involved in a moderation action.

I will say that there have been times where the process didn't work the way I liked - but I am unable to discuss any of it here (except with TSC members) due to privacy/confidentiality.

Reply:
+1 to @benjamingr comment.